tests/python_package_test/test_dask.py

```diff
@@ -74,15 +178,14 @@ def test_classifier(output, centers, client, listen_port):
     dask_classifier = dlgbm.DaskLGBMClassifier(time_out=5, local_listen_port=listen_port)
     dask_classifier = dask_classifier.fit(dX, dy, sample_weight=dw, client=client)
     p1 = dask_classifier.predict(dX)
-    s1 = accuracy_score(dy, p1)
+    s1 = da.average(dy == p1).compute()
```

Comment:
why did this test have to change? If it's because of something nondeterministic in `accuracy_score()` or some precision problem that makes this test flaky, could you make that a separate PR?

Reply:
Nope, had to do with eliminating the `dask-ml` dependency.

I saw that `dask-ml` was missing from setup.py, so I just tried to remove that dependency since we're only using it for two `sklearn.metrics`-emulating functions anyway (`accuracy_score` and `r2_score`). Since dask-ml is now part of test.sh I can just keep this as `accuracy_score`.

Reply:
ok cool, thank you

Reply:
Addressed in 78358e2